services/web/server/src/simcore_service_webserver/trash/_rest.py

```diff
@@ -65,4 +58,10 @@ async def empty_trash(request: web.Request):
         fire_and_forget_tasks_collection=request.app[APP_FIRE_AND_FORGET_TASKS_KEY],
     )
 
+    # NOTE: Ensures `fire_and_forget_task` is triggered; otherwise,
+    # when the front-end requests the trash item list,
+    # it may still display items, misleading the user into
+    # thinking the `empty trash` operation failed.
+    await asyncio.sleep(1)
```

Comment:
Q: does `sleep(0)` not work as well?

Reply:
yes, that would trigger the task but I want it to go at least on second in it

Reply:
this is a bit shaky... I would advocate for an asyncio Event instead of some sleep witha random number... Please at least use a constant then.

Reply:
good idea! thx